jekyll/_cci2/runner-faqs-adoc

```diff
@@ -0,0 +1,58 @@
+= CircleCI Runner FAQs
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+This page answers frequently asked questions for the CircleCI Runner product.
+
+toc::[]
+
+=== What is the pricing for the CircleCI runner?
+
+Currently, CircleCI does not charge for runner usage; however, this will change in the future. If you are a "preview" customer, your account team will keep you informed of any pricing changes for the runner.
```

Comment:
This is no longer true. We should remove the pricing section for now, we can re-add it and point to the Scale plan once the relevant marketing page is up.

Reply:
Removed from document